packages/runtime/src/account.ts

```diff
@@ -356,7 +355,7 @@ class Asset {
     this.id = assetId;
     this.definitions = {
       creator: creator,
-      total: def.total,
+      total: BigInt(def.total), // ........> ....> ....> ....> ....> ....> ....> ....> ....> ....> ....> ....> ....> ....> ....>....>
```

Comment:
what is this comment?

Reply:
Sorry, that shouldn't be here :sweat_smile: